extensions/blocks/instagram-gallery/editor.scss

```diff
@@ -1,5 +1,18 @@
 @import './view.scss';
 
+.wp-block-jetpack-instagram-gallery .components-placeholder {
+	.components-radio-control {
+		margin-bottom: $default-block-margin;
+		label {
+			font-weight: normal;
```

Comment:
I tried removing this and it looks the same. Is it for a different version of Gutenberg?

Reply:
Interesting!
If I remove that line, it looks like this to me:

<img width="613" alt="Screenshot 2020-05-21 at 17 49 03" src="https://user-images.githubusercontent.com/2070010/82583777-6d9d3480-9b8b-11ea-8fb7-7f502f01308f.png">

I've tested on Guten 8.1 and 8.0, and with Guten deactivated, and in all cases I see this style in the dev tools:
```css
.editor-styles-wrapper label {
  font-weight: 600;
}
```

(Actually, I'm not entirely sure I tested on 8.0. I just updated Guten this morning after a while, and it's 8.1 now. Maybe it was a 8.1 beta before.)

Reply:
Maybe it's a theme thing? Doesn't hurt to leave it in :)